iothub/device/src/Fx/Fx.cs

```diff
@@ -44,15 +43,18 @@ public static bool IsFatal(Exception ex)
             while (ex != null)
             {
                 // FYI, CallbackException is-a FatalException
```

Comment:
why we have this comment here but no include `CallbackException` within this method?

Reply:
Sounds like something worth removing.